react/components/molecules/snackbar-ios/snackbar-ios.js

```diff
@@ -0,0 +1,125 @@
+import React, { PureComponent } from "react";
+import { Animated, Platform, StyleSheet, Text, ViewPropTypes } from "react-native";
+import PropTypes from "prop-types";
+import { mix } from "yonius";
+
+import { baseStyles, IdentifiableMixin } from "../../../util";
+
+import { Touchable } from "../../atoms";
+
+export class SnackbarIos extends mix(PureComponent).with(IdentifiableMixin) {
+    static get propTypes() {
+        return {
+            text: PropTypes.string,
+            actionText: PropTypes.string,
+            duration: PropTypes.number,
+            animationDuration: PropTypes.number,
+            onActionPress: PropTypes.func,
+            style: ViewPropTypes.style
+        };
+    }
+
+    static get defaultProps() {
+        return {
+            text: undefined,
+            actionText: undefined,
+            duration: 5000,

```

Comment:
maybe rename to `showDuration`? It's difficult to understand what is the duration of.